app/wama.web/Views/AdminConsole/UserTool/ViewAccount.cshtml

```diff
@@ -15,6 +15,8 @@
     {
         string MemberID = ViewBag.MemberId;
         <div class="col-sm-8 col-sm-offset-2 col-xs-10 col-xs-offset-1">
+            @if (Model.AccountType == UserAccountType.Patron)
```

Comment:
You guys have to stop the excuse of, "**Dubem told me to**".

We're all part of this project, hence should understand the requirements.